src/NLog.Targets.Syslog/MessageSend/Tcp.cs

```diff
@@ -104,7 +104,11 @@ private Stream SslDecorate(TcpClient tcpClient)
 
             // Do not dispose TcpClient inner stream when disposing SslStream (TcpClient disposes it)
             var sslStream = new SslStream(tcpStream, true);
+#if NETSTANDARD1_3 || NETSTANDARD1_5
+            sslStream.AuthenticateAsClientAsync(Server, retrieveClientCertificates(), SslProtocols.Tls12, false).Wait();
+#else
             sslStream.AuthenticateAsClient(Server, retrieveClientCertificates(), SslProtocols.Tls12, false);
+#endif
 
```

Comment:
Well all sync-methods in Net Core has been removed, so that is why the awkward way (Because the Syslog-code expects it to be sync)